### PR TITLE
011 data structure for keywords and synonyms 

### LINKED ISSUE #11: Create data structure for keywords and synonyms
A data structure containing a sequence of synonyms per keyword

### PR DESCRIPTION
**Pull request by irinakulinich3712**
This data structure consists of keywords from the park data as keys and vectors of their synonyms as values

closes #11

### CODE REVIEW

**Comment by adzai** [comment]
#21

File: data/synonyms.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+"skating" : ["skating", "roller", "skateboard", "rollerblade"],
+"sports" : ["sports", "gymnastics", "exercise"]
+"playground" : ["playground", "playing"],
+"transportation" : ["transportation", "metro", "subway", "tram", "bus", "transport"],
+"parking" : ["parking", "car"],
+"opening" : ["opening", "open", "enter"]
+}
+
+
+
```
Comment (adzai) [inline data/synonyms.json:15]: Cleanup the newlines and I think this can be merged.

Comment (irinakulinich3712) [reply data/synonyms.json]: Done

Comment (adzai) [reply data/synonyms.json]: Go ahead and merge this if it's ready

**Review by AniSanikidze — COMMENTED** [review]
In the data-en.json file, I removed the "opening" field as we won't provide detailed information about the exact opening hours. So  I think saying that the park is open is not necessary. The same goes for transportation and attraction fields but I didn't remove them.  I think, for now, we should wait for Petr's answer and based on that update the data-en.json and synonyms.json.  Other than that, it looks good for me.

File: data/synonyms.json (hunk 1 above)
Comment (AniSanikidze) [reply data/synonyms.json]: I think, for now, we should wait for Petr's answer regarding having fields like "transportation", "open" and "attractions", for which only having yes/no values and not providing detailed information don't really make sense.  let's wait for Petr's feedback  if we should leave these fields as they are right now  or to update them.  Also, I think it's better to first merge data-en.json file and then synonyms as they should correspond to each other.

**Review by AniSanikidze — COMMENTED** [review]
Also, for some parks there is  a field "skiing" which is missed here.

**Comment by irinakulinich3712** [comment]
> In the data-en.json file, I removed the "opening" field as we won't provide detailed information about the exact opening hours. So I think saying that the park is open is not necessary. The same goes for transportation and attraction fields but I didn't remove them. I think, for now, we should wait for Petr's answer and based on that update the data-en.json and synonyms.json. Other than that, it looks good for me.

Ok, I will remove the "opening" field

**Comment by irinakulinich3712** [comment]
> Also, for some parks there is a field "skiing" which is missed here.

Oh, I haven't seen that field. In the park data I have only found one instance of it. But this field the should be all other records then, just with the "no" value

**Pull request merged**